devtools/server/actors/root.js

```diff
@@ -324,6 +330,38 @@ RootActor.prototype = {
     });
   },
 
+  onGetWindow: function ({ outerWindowID }) {
+    if (!DebuggerServer.allowChromeProcess) {
+      return {
+        from: this.actorID,
+        error: "forbidden",
+        message: "You are not allowed to debug windows."
+      };
+    }
+    let window = Services.wm.getOuterWindowWithId(outerWindowID);
+    if (!window) {
```

Comment:
Nit: maybe just `if (!Services.wm.getOuterWindowWithId(outerWindowID))`. Otherwise, change `let` to `const` to signal it won't change.